holmes/plugins/toolsets/aks-node-health.yaml

```diff
@@ -13,43 +13,46 @@ toolsets:
         user_description: "get the status of all nodes in the AKS cluster"
         command: |
           kubectl get nodes
-
       - name: "describe_node"
         description: "Describes a specific node in the AKS cluster to inspect its conditions."
         user_description: "describe node {{ NODE_NAME }} in the AKS cluster"
         command: |
           kubectl describe node {{ NODE_NAME }}
-
       - name: "check_kubelet_logs"
-        description: "Fetches kubelet logs from a specific node in the AKS cluster."
-        user_description: "get kubelet logs for node {{ NODE_NAME }}"
-        command: |
-          ssh {{ NODE_NAME }} 'journalctl -u kubelet'
-
-      - name: "check_control_plane_connectivity"
-        description: "Verifies if the node can communicate with the control plane."
-        user_description: "check control plane connectivity for node {{ NODE_NAME }}"
+        description: "Fetches kubelet logs from a specific node in the AKS cluster using Azure CLI VMSS run command."
+        user_description: "get kubelet logs for node {{ NODE_NAME }} in the AKS cluster"
```

Comment:
Should the parameters in the `user description` match the command parameters ?

Reply:
just to clarify, since the `NODE_NAME` is in the user description, but not in the command parameters, it might appear empty when describing the command